client/extensions/woocommerce/state/ui/products/variations/edits-reducer.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * External dependencies
+ */
+import { isNumber } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import {
+	WOOCOMMERCE_EDIT_PRODUCT_VARIATION,
+} from '../../../action-types';
+
+const initialState = null;
+
+export default function( state = initialState, action ) {
+	const handlers = {
+		[ WOOCOMMERCE_EDIT_PRODUCT_VARIATION ]: editProductVariationAction,
+	};
+
+	const handler = handlers[ action.type ];
+
+	return ( handler && handler( state, action ) ) || state;
+}
+
+function editProductVariationAction( edits, action ) {
+	const { product, variation, data } = action.payload;
+	const prevEdits = edits || [];
+	const productId = product.id;
+	const bucket = variation && isNumber( variation.id ) && 'updates' || 'creates';
+
+	let found = false;
+
+	// Look for an existing product edits first.
+	const _edits = prevEdits.map( ( productEdits ) => {
```

Comment:
We started adopting the underscores for the new versions of things that are destined to replace the old ones.